### PR TITLE
New donor item for Cuackles: singularity hat

### PR DESCRIPTION
**Pull request by cuackles**
# Document the changes in your pull request

adds a singulo hat for me, cus i wanna be cool like cowbot and mqiib
image
hatgulo
singulo hat

# Changelog
:cl:  
rscadd: new donor item, hatgulo for me
imageadd: added a new donor hat sprite for myself
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
NO!!!!!!!!!!!!!!!! NO MORE DONOR TIEMS!!!!!!!!!!!!!

**Comment by cuackles** [comment]
> NO!!!!!!!!!!!!!!!! NO MORE DONOR TIEMS!!!!!!!!!!!!!

BUT DAAAAAD ALL MY FRIENDS HAVE ONEEEEEEE I WANT ONE TOOOOOO!!!!!

**Comment by Mqiib** [comment]
my IP is being stolen i want a cut of the profits

**Comment by MajManatee** [comment]
i will make myself a hat too if this is merged

**Comment by cuackles** [comment]
> my IP is being stolen i want a cut of the profits

I will let you wear the hat if it gets merged

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -349,3 +349,8 @@
 	icon_state = "cowboy"
 	item_state = "cowboy"
 
+/obj/item/clothing/head/hatgulo
+	name = "Singularity hat"
+	desc = "How did they manage to put it inside the hat, only the Wizard federation knows"
```
Comment (Anvilman6) [inline code/modules/clothing/head/misc_special.dm]:
```suggestion
	desc = "How does a singularity fit in a hat? Only the Wizard Federation knows..."
```

**Review by Anvilman6 — COMMENTED** [review]
Not like it really matters, since this is a donor item, but: suggested description tweak

File: code/modules/clothing/head/misc_special.dm (hunk 1 above)
Comment (cuackles) [reply code/modules/clothing/head/misc_special.dm]: quality always  matters i would say

**Comment by nesisknesis** [comment]
The shading is pretty shoddy and black outlines on a sprite that is not black does not look nice. Hard to describe the animation but it looks uncanny.

**Pull request closed without merging**